.buildkite/build-test-omnibus.sh

```diff
@@ -21,92 +28,262 @@ for platform in ${platforms[@]}; do
 done
 
 # remove duplicates from build array
-omnibus_build_platforms=($(printf "%s\n" "${omnibus_build_platforms[@]}" | sort -u | tr '\n' ' '))
+if [[ ! -z "${omnibus_build_platforms:-}" ]]
```

Comment:
I think `-z` should handle an unset variable comparison, but also quotes should force to a zero length string, so the `:-` feels redundant?